packages/react/src/Link/Link.tsx

```diff
@@ -53,6 +53,7 @@ const Link = forwardRef(({as: Component = 'a', className, inline, underline, ...
         data-muted={props.muted}
         data-inline={inline}
         data-underline={underline}
+        data-hover-color={hoverColor}
```

Comment:
does this need a css counterpart or does it just work? 🤔

Reply:
I don't believe it's actually being used any longer since it is deperecated, this just ensures that we don't pass in an invalid attribute into the dom

Reply:
I see, thanks for explaining that!
Do we gain anything by adding the `data-hover-holor` then? could we just ignore the prop?